Mathlib/CategoryTheory/FiberedCategory/Fiber.lean

```diff
@@ -107,17 +110,22 @@ def inducedFunctor : C ⥤ Fiber p S where
   map φ := ⟨F.map φ, of_commsq _ _ _ (congr_obj hF _) (congr_obj hF _) <|
     by simpa using (eqToIso hF).hom.naturality φ⟩
 
-@[simp]
-lemma inducedFunctor_map {X Y : C} (f : X ⟶ Y) :
-    fiberInclusion.map ((inducedFunctor hF).map f) = F.map f := rfl
-
 /-- Given a functor `F : C ⥤ 𝒳` such that `F ⋙ p` is constant at some `S : 𝒮`, then
 we get a natural isomorphism between `inducedFunctor _ ⋙ fiberInclusion` and `F`. -/
 @[simps!]
-def inducedFunctorCompIsoSelf : (inducedFunctor hF) ⋙ fiberInclusion ≅ F := Iso.refl _
+def inducedFunctorCompIsoSelf : (inducedFunctor hF) ⋙ fiberInclusion ≅ F := .refl _
 
+@[simp]
```

Comment:
I believe we should refrain from making this a simp lemma.

Reply:
Could you explain why? To me, this seems like a substitution one would want to do whenever possible.

Reply:
I do not have issues with the `obj/map`-applied versions, but equalities of functors are a little bit controversial. I understand that in some proofs, it would be convenient to do the substitution, but I am more reluctant to have this as a simp lemma. (For example, `𝟭 _ ⋙ F = F` is not a simp lemma.)

Reply:
Ahh good point!